File: GNodeBGen/src/main/java/com/ericsson/tcs/topology/wrapper/NodeWrapper.java
package com.ericsson.tcs.topology.wrapper;

import java.util.Map;

import com.google.gson.annotations.SerializedName;

public class NodeWrapper {

	@SerializedName("_NodeWrapper__managedElement")
	private String __managedElement;

	@SerializedName("_NodeWrapper__networkFnDetailsMap")
	private Map<String, NetworkFunctionWrapper> __networkFnDetailsMap;

	/**
	 * @return the __managedElement
	 */
	public String get__managedElement() {
		return __managedElement;
	}

	/**
	 * @param __managedElement
	 *            the __managedElement to set
	 */
	public void set__managedElement(String __managedElement) {
		this.__managedElement = __managedElement;
	}

	/**
	 * @return the __networkFnDetailsMap
	 */
	public Map<String, NetworkFunctionWrapper> get__networkFnDetailsMap() {
		return __networkFnDetailsMap;
	}

	/**
	 * @param __networkFnDetailsMap
	 *            the __networkFnDetailsMap to set
	 */
	public void set__networkFnDetailsMap(Map<String, NetworkFunctionWrapper> __networkFnDetailsMap) {
		this.__networkFnDetailsMap = __networkFnDetailsMap;
	}

}

File: GNodeBGen/src/main/resources/bin/nodeWrapper.py
#!/usr/bin/python


class NodeWrapper():

    __managedElement = None
    __networkFnDetailsMap = None
    __listOfNci = []
    __uniqueNciCount = 0
    
    def get_unique_nci_count(self):
        return self.__uniqueNciCount

    def set_unique_nci_count(self):
        self.__uniqueNciCount = len(self.__listOfNci)

    def get_list_of_nci(self):
        return self.__listOfNci

    def set_list_of_nci(self, value):
        self.__listOfNci = value

    def get_managed_element(self):
        return self.__managedElement

    def get_network_fn_details_map(self):
        return self.__networkFnDetailsMap

    def set_managed_element(self, value):
        self.__managedElement = value

    def set_network_fn_details_map(self, value):
        self.__networkFnDetailsMap = value

